packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -75,6 +92,77 @@ export default function Sidebar({
     return <></>;
   }
 
+  if (useNewMobileLayout && !isLaptop && !isMobile) {
+    const buttonProps: ButtonProps<'a' | 'button'> = {
+      variant: ButtonVariant.Tertiary,
+      style: { width: '100%' },
```

Comment:
Why don't we just use tailwind?

Reply:
copy/pasta from here: https://github.com/dailydotdev/apps/blob/34863591dc2dcb5066107ea19957e5fe98d9a885/packages/webapp/components/FooterNavBar.tsx#L106-L110

I'll fix in both places